app/components/funding-progress.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from "./ui/button";
```

Comment:
can move this + `report-sidebar.tsx` + `report-support-feed.tsx` into new folder `components/report-detail`